core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -284,8 +289,8 @@ private void handleHttp2Settings(ChannelHandlerContext ctx, Http2Settings h2sett
     private ServerHttp2ObjectEncoder newServerHttp2ObjectEncoder(ChannelHandlerContext ctx,
                                                                  Http2ServerConnectionHandler handler) {
         return new ServerHttp2ObjectEncoder(ctx, handler.encoder(), handler.keepAliveHandler(),
-                                            config.isDateHeaderEnabled(),
-                                            config.isServerHeaderEnabled()
+                                            configHolder.getConfig().isDateHeaderEnabled(),
```

Comment:
Is there any reason that you used `configHolder.getConfig()` here?
We are using just `config` for other places in this class.

Reply:
you are right. I think I missed this code. Will revert it back to config instead of configHolder.getConfig()